datasync/downloader/queue_test.go

```diff
@@ -127,14 +133,20 @@ func TestBasics(t *testing.T) {
 			// queue size is only 10, so throttling should occur
 			t.Fatal("should throttle")
 		}
-		// But we should still get the first things to fetch
+		// But we should still get the first things to fetch.
```

Comment:
Period is used only here. I think we don't need this since it lowers the consistency.

Reply:
Applied. Thanks.